src/Uno.UWP/UI/ViewManagement/ApplicationView.cs

```diff
@@ -91,6 +95,31 @@ public ApplicationViewMode ViewMode
 			}
 		}
 
+		public static Size PreferredLaunchViewSize
+		{
+			get
+			{
+				if (ApplicationData.Current.LocalSettings.Values.TryGetValue(PreferredLaunchViewWidthKey, out var widthObject) &&
+					ApplicationData.Current.LocalSettings.Values.TryGetValue(PreferredLaunchViewHeightKey, out var heightObject))
+				{
+					double width = (double)widthObject;
+					double height = (double)heightObject;
```

Comment:
It would be best to use a safe cast here (`widthObject is double width`), just in case the persisted values get corrupted somehow.